README.md

```diff
@@ -697,7 +697,22 @@ Devise supports ActiveRecord (default) and Mongoid. To select another ORM, simpl
 
 ### Rails API Mode
 
-Rails 5+ has a built-in [API Mode](https://edgeguides.rubyonrails.org/api_app.html) which optimizes Rails for use as an API (only). One of the side effects is that it changes the order of the middleware stack, and this can cause problems for `Devise::Test::IntegrationHelpers`. This problem usually surfaces as an ```undefined method `[]=' for nil:NilClass``` error when using integration test helpers, such as `#sign_in`. The solution is simply to reorder the middlewares by adding the following to test.rb:
+Rails 5+ has a built-in [API Mode](https://edgeguides.rubyonrails.org/api_app.html) which optimizes Rails for use as an API (only). This comes with a set of side effects for what devise is able to support without additional modifications and some issues you may face during development/testing.
+
+#### Supported Authentication Strategies
+The only default configured strategy supported by devise for API-only applications is database via HTTP Auth, meaning the user is effectively authenicated on each request (a byproduct of API-only applications not supporting browser-based sessions via cookies).
+
+The devise default for HTTP Auth is disabled, so it will need to be enabled in the devise initializer for the database strategy:
+
+```ruby
+config.http_authenticatable = [:database]
+```
+
+This restriction does not limit you from implementing custom strategies, either in your application or via gem-based extensions for devise.
+A common authentication strategy for APIs is token-based authenication. For more information on extending devise to support this type of authentication and others, see the wiki article for [Simple Token Authentication Examples and alternatives](https://github.com/plataformatec/devise/wiki/How-To:-Simple-Token-Authentication-Example#alternatives).
```

Comment:
Little typo 😄
But I also think that we can link [@tegon's article](http://blog.plataformatec.com.br/2019/01/custom-authentication-methods-with-devise/) about adding custom authentication strategies here. Something like:

```suggestion
A common authentication strategy for APIs is token-based authentication. For more information on extending devise to support this type of authentication and others, see the wiki article for [Simple Token Authentication Examples and alternatives](https://github.com/plataformatec/devise/wiki/How-To:-Simple-Token-Authentication-Example#alternatives) or the this blog post on [Custom authentication methods with Devise](http://blog.plataformatec.com.br/2019/01/custom-authentication-methods-with-devise/).
```